### PR TITLE
Speedrunning Mixtral

### PR DESCRIPTION
**Pull request by pcuenca**

### CODE REVIEW

File: _blog.yml [hunk 1]
```diff
@@ -3169,3 +3169,15 @@
     - nlp
     - llm
     - guide
+
+- local: mixtral
+  title: "Welcome Mixtral - a SOTA Mixture of Experts on Hugging Face"
+  author: lewtun
+  thumbnail: /blog/assets/mixtral/thumbnail.png
```
Comment (lewtun) [inline _blog.yml:3176]: i quite like this image for the thumbnail

image (9)

File: mixtral.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+- Outperforms Llama 2 70B and matches or beats GPT3.5 on most benchmarks
+- Speaks English, French, German, Spanish, and Italian.
+- Good at coding, with 40.2% on HumanEval
+- Commercially permissive with an Apache 2.0 license
+
+So how good are the Mixtral models? Here’s an overview of the base model and its performance compared to other open models on the [LLM Leaderboard](https://huggingface.co/spaces/HuggingFaceH4/open_llm_leaderboard) (higher scores are better):
+
+| Model | License | Commercial use? | Pretraining size [tokens] | Leaderboard  score ⬇️  |
+| --- | --- | --- | --- | --- |
+| https://huggingface.co/mistralai/Mixtral-8x7B-v0.1 | Apache 2.0 | ✅ | unknown |  |
```
Comment (philschmid) [inline mixtral.md:45]: TODO: Add evals when available

File: mixtral.md [hunk 3]
```diff
@@ -202,14 +216,14 @@ This takes about 9 hours to train on a single A100, but can be easily parallelis
...
 - [Mixture of Experts Explained](https://huggingface.co/blog/moe)
 - [Mixtral of experts](https://mistral.ai/news/mixtral-of-experts/)
 - [Models on the Hub](https://huggingface.co/models?other=mixtral)
 - [Open LLM Leaderboard](https://huggingface.co/spaces/HuggingFaceH4/open_llm_leaderboard)
 - [Chat demo on Hugging Chat](https://huggingface.co/chat/?model=mistralai/Mixtral-8x7B-Instruct-v0.1)
 
-## **Conclusion**
+## Conclusion
 
 We're very excited about Mixtral being released! In the coming days, be ready to learn more about ways to run your own fine-tuning, execute the smallest models on-device, and many other exciting updates we're preparing for you!
```
Comment (philschmid) [inline mixtral.md:229]:
```suggestion
We're very excited about Mixtral being released! In the coming days, be ready to learn more about ways to fine-tune and deploy Mixtral. 
```

File: mixtral.md [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+
+```bash
+<s> [INST] User Instruction 1 [/INST] Model answer 1</s> [INST] User instruction 2[/INST]
+```
+
+This format has to be exactly reproduced for effective use. We’ll show later how easy it is to reproduce the instruct prompt with the chat template available in `transformers`. 
+
+### What we don't know
+
+Like the previous Mistral 7B release, there are several open questions about this new series of models. In particular, for pertaining, we have no information about the size of the dataset, its composition, or how it was preprocessed.
```
Comment (pcuenca) [inline mixtral.md:96]:
```suggestion
Like the previous Mistral 7B release, there are several open questions about this new series of models. In particular, we have no information about the size of the dataset used for pretraining, its composition, or how it was preprocessed.
```

File: mixtral.md [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+## Demo
+
+You can chat with the Mixtral Instruct model on Hugging Face Chat! Check it out here: [https://huggingface.co/chat/?model=mistralai/Mixtral-8x7B-Instruct-v0.1](https://huggingface.co/chat/?model=mistralai/Mixtral-8x7B-Instruct-v0.1).
+
+## Inference
+
+We provide two main ways to run inference with Mixtral models:
+
+- Via the `pipeline()` function of 🤗 Transformers.
+- With Text Generation Inference, which supports advanced features like continuous batching, tensor paralleism and more for blazing fast results.
```
Comment (pcuenca) [inline mixtral.md:109]:
```suggestion
- With Text Generation Inference, which supports advanced features like continuous batching, tensor parallelism, and more, for blazing fast results.
```

File: mixtral.md [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+	--max-total-tokens 32000
+```
+
+## Fine-tuning with 🤗 TRL
+
+Training LLMs can be technically and computationally challenging. In this section, we look at the tools available in the Hugging Face ecosystem to efficiently train Mixtral on a single A100 GPU.
+
+An example command to fine-tune Mixtral on OpenAssistant’s [chat dataset](https://huggingface.co/datasets/OpenAssistant/oasst_top1_2023-08-25) can be found below. To conserve memory, we make use of 4-bit quantization and [QLoRA](https://arxiv.org/abs/2305.14314) to target all the linear layers in the attention blocks. Note that unlike dense transformers, one should not target the MLP layers as they are sparse and don’t interact well with PEFT.
+
+First install the nightly version of 🤗 TRL and clone the repo to access the [training script](https://github.com/huggingface/trl/blob/main/examples/scripts/sft.py):
```
Comment (pcuenca) [inline mixtral.md:191]:
```suggestion
First, install the nightly version of 🤗 TRL and clone the repo to access the [training script](https://github.com/huggingface/trl/blob/main/examples/scripts/sft.py):
```

**Review by philschmid — APPROVED** [review]

**Pull request merged**